src/main/java/org/icgc/argo/workflow_management/wes/NextflowService.java

```diff
@@ -65,11 +65,12 @@ public class NextflowService implements WorkflowExecutionService {
 
   // Dependencies
   private final NextflowProperties config;
+
   private final SecretProvider secretProvider;
   private final WebLogEventSender webLogSender;
 
   // State
-  private final DefaultKubernetesClient workflowRunK8sClient;
+  private DefaultKubernetesClient workflowRunK8sClient;
```

Comment:
For my own knowledge, why is this no longer set as `final`?

Reply:
as it is not a part of the constructor anymore.